src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -63,7 +66,7 @@ public void TestInitialize()
         public void Ctor_WhenGivenANullActiveSolutionBoundTracker_ThrowsArgumentNullException()
```

Comment:
I think we should delete most, if not all, of the `ArgumentNullException` tests. They can't be null in a MEF ctor.

Reply:
Agreed for the MEF tests. We should remove the null checks too, or change them to assertions.